docs/sphinx/reference-libobs-util-darray.rst

```diff
@@ -163,13 +164,14 @@ dynamic array value with a reference (&) operator.  For example:
 
 ---------------------
 
-.. function:: void da_push_back_array(da, const void *src_array, size_t item_count)
+.. function:: size_t da_push_back_array(da, const void *src_array, size_t item_count)
 
    Pushes an array of values to the back of the array.
 
    :param da:         The dynamic array
    :param src_array:  Pointer of the array of values
    :param item_count: Number of items to push back
+   :return:           Index of the new value
```

Comment:
I think I’d go with "Index of the first new value" here as there are multiple new values added.

Reply:
Fixed, thanks!